packages/concerto-vocabulary/test/__snapshots__/vocabularymanager.js.snap

```diff
@@ -127,10 +307,30 @@ Object {
         "arguments": Array [
           Object {
             "$class": "concerto.metamodel@1.0.0.DecoratorString",
-            "value": undefined,
+            "value": "VIN",
           },
         ],
-        "name": "Term_.description",
+        "name": "Term_.tooltip",
```

Comment:
Can we make this `Term.tooltip` ?

Reply:
```
namespace org.acme

@Term("A color")
enum Color {
  @Term("RED of the Color")
  o RED
  @Term("BLUE of the Color")
  o BLUE
  @Term("GREEN of the Color")
  o GREEN
}

@Term("A road vehicle")
asset Vehicle {
  @Term("Vehicle Identification Number")
  @Term.tooltip("VIN")
  o String vin
  @Term("Color of the Vehicle")
  o String color
  @Term("Model of the vehicle")
  @Term.description("The weight of the truck in KG")
  @Term.tooltip("VIN")
  o String model
}

@Term("A truck")
@Term.description("A vehicle capable of carrying cargo")
@Term.tooltip("Truck")
asset Truck {
  @Term("The weight of the truck")
  @Term.tooltip("Truck weight")
  @Term.description("The weight of the truck in KG")
  o Double weight
}
```
Something like this?

Reply:
Actually, I'm not sure '.' in a decorator name is valid and will parse. We may have to use underscore...

Reply:
Can you confirm this? Which once should we use? `@Term.tooltip("Truck weight")` or `@Term_tooltip("Truck weight")`